app/models/metasploit_data_models/module_run.rb

```diff
@@ -0,0 +1,194 @@
+# {MetasploitDataModels::ModuleRun} holds the record of having launched a piece of Metasploit content.
+# It has associations to {Mdm::User} for audit purposes, and makes polymorphic associations to things like
+# {Mdm::Vuln} and {Mdm::Host} for flexible record keeping about activity attacking either specific vulns or just
+# making mischief on specific remote targets w/out the context of a vuln or even a remote IP service.
+#
+# There are also associations to {Mdm::Session} for two use cases: a `spawned_session` is a
+# session created by the ModuleRun. A `target_session` is a session that the ModuleRun
+# is acting upon (e.g.) for running a post module.
+class MetasploitDataModels::ModuleRun < ActiveRecord::Base
+  #
+  # Constants
+  #
+
+  # Marks the module as having successfully run
```

Comment:
My habit for constants now has been to put them in their our ruby `Module` named after the attribute they are valid for.  So this would move to as follows:

```ruby
module MetasploitDataModels::ModuleRun::Status
  #
  # CONSTANTS
  #
  
  # Marks the module as having had a runtime error
  ERROR       = 'error'
  # Marks the run as having not run successfully
  FAIL        = 'failed'
  # Marks the module as having successfully run
  SUCCEED     = 'succeeded'

  # {ModuleRun} objects will be validated against these statuses
  ALL = [
    ERROR,
    FAIL,
    SUCCEED
  ]
end
```

I also alphabetize the constants.  Finally, by convention, I use ALL as the constant with the list of the individual constants and use the ALL for the validates inclusion argument.